Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -4931,6 +4932,201 @@ RefPtr<StyleValue> Parser::parse_content_value(Vector<ComponentValue> const& com
     return ContentStyleValue::create(StyleValueList::create(move(content_values), StyleValueList::Separator::Space), move(alt_text));
 }
 
+// https://w3c.github.io/csswg-drafts/css-display/#the-display-properties
```

Comment:
Annoyingly there are several different places that specs get hosted. Try to use the w3.org one if it is available and up to date with what you're implementing, since it's more official (and more stable).

```suggestion
// https://www.w3.org/TR/css-display-3/#the-display-properties
```